modules/oc-mirror-enclave-support.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: Procedure
+[id="oc-mirror-enclave-support_{context}"]
+== How to mirror to an enclave
+
+When you mirror to an enclave, you must first transfer the necessary images from one or more enclaves into the enterprise central registry.
+
+The central registry is situated within a secure network, specifically a disconnected environment, and is not directly linked to the public internet. But the user must execute `oc mirror` in an environment with access to the public internet.
+
+.Procedure
+
+. Before running oc-mirror plugin v2 in the disconnected environment, create a `registries.conf` file.
++
+It is recommended to store the file under `$HOME/.config/containers/registries.conf` or `/etc/containers/registries.conf`.
++
+The TOML format of the file is described in this specification.
```

Comment:
Maybe group all these paragraphs into one so that step 1 is not overly padded with line breaks?

Reply:
Group into one step and add the recommended step as a note?